debezium-core/src/main/java/io/debezium/rest/ConnectionValidationResource.java

```diff
@@ -27,7 +27,7 @@ default ValidationResults validateConnectionProperties(Map<String, ?> properties
         ValidationResults validationResults;
         ClassLoader originalClassLoader = Thread.currentThread().getContextClassLoader();
         Thread.currentThread().setContextClassLoader(getConnector().getClass().getClassLoader());
```

Comment:
@rk3rn3r  BTW is this still required now when the actual rest resource is in the connector module? I'm just curious.

Reply:
The reason for this is that JAX RS resources for Connect REST extensions are loaded in an isolated way with an isolated class loader that has no access to anything else except of the exposed interfaces. I need to "break" this.